charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml

```diff
@@ -364,7 +358,7 @@ data:
         LabelMapPath /fluent-bit/etc/kubernetes_label_map.json
         DynamicHostPath {"kubernetes": {"namespace_name": "namespace"}}
         DynamicHostPrefix http://loki.
-        DynamicHostSulfix .svc:3100/loki/api/v1/push
+        DynamicHostSuffix .svc:3100/loki/api/v1/push
```

Comment:
This will not work with the current fluent-bit version.
Was a typo which this huge PR of @vlvasilev is fixing: https://github.com/gardener/logging/pull/64

Reply:
hehe, ok, I'll revert it. It was a best guess anyways :)

Reply:
I reverted it, PTAL